routes/root.js

```diff
@@ -12,136 +13,216 @@ const {
 router.use(csrfProtection);
 
 // Home route
-router.get('/', function(req, res) {
-  const routeReady = () => {
-    if (recents && drafted && blog && decks && featured) {
-      decklinks = decks.splice(Math.max(decks.length - 10, 0), decks.length);
-      res.render('index', {
-        devblog: blog.length > 0 ? blog[0] : null,
-        recents: recents,
-        drafted: drafted,
-        decks: decklinks,
-        featured: featured
-      });
-    }
-  };
-
-  var user_id = '';
-  var recents, drafted, blog, decks, featured;
+router.get('/', async function(req, res) {
+  req.user ? res.redirect('/dashboard') : res.redirect('/landing');
+});
 
-  if (req.user) user_id = req.user._id;
-  Cube.find({
-    $or: [{
-        $and: [{
-          'card_count': {
-            $gt: 200
-          }
-        }, {
-          'isListed': true
-        }]
-      },
-      {
+router.get('/explore', async function(req, res) {
+  const user_id = req.user ? req.user._id : '';
+
+  [recents, featured, drafted, blog, decks] = await Promise.all([
```

Comment:
Style nit-pick to match style in dashboard route, etc:
query1 = Cube.find(...);
query1 = Cube.find(...);
const [result1, result2] = await Promise.all([query1, query2];

Reply:
Yes, I was meaning to make this change. This was my first time switching a route to use Promise.all() and a comment there recommended me to create vars for each query to improve readability.